app/jobs/deposit_job.rb

```diff
@@ -22,6 +22,7 @@ def perform(work_form:, work:, deposit:)
     work.update!(deposit_job_started_at: nil, druid:)
     # Just to be aware for future troubleshooting: There is a possible race condition between the websocket
     # connecting and the following broadcast being sent.
+    sleep 0.5 if Rails.env.test? # Avoids race condition in tests
```

Comment:
I wonder if there is a way to do this without `Rails.env.test?` being in the job code. Not a blocker, just something to think about.